source/Grid/Grid.js

```diff
@@ -204,6 +204,8 @@ export default class Grid extends Component {
 
     // See defaultCellRangeRenderer() for more information on the usage of this cache
     this._cellCache = {}
+
+    this._scrollbarSize = getScrollbarSize()
```

Comment:
@bvaughn  I think this is not good. Because the first render will not be correct. May be we can skip set the overflow style in first render? I think this will not affect #116.